test/api/test_invoice.py

```diff
@@ -204,6 +204,55 @@
 }
 """
 
+retrieveInvoiceExample = """
+{
+  "uuid": "inv_123",
```

Comment:
Again, I'm somewhat surprised that this doesn't cause errors, I guess we must not be actually parsing the UUID from a string anywhere.